jobsapp/api/permissions.py

```diff
@@ -6,9 +6,5 @@
 
 class IsEmployer(BasePermission):
     def has_permission(self, request: Request, view: View) -> bool:
-        return request.user and request.user.role == "employer"
+        return request.user
```

Comment:
Method must response a bool.  My opinion is you can remove all `has_permission` method from code.

Reply:
@victormartingarcia I think if we remove the `role` field from user model, we don't need anymore this permission. Neither the `IsEmployee`. Right @francescarpi ?

Reply:
Exactly. I think the same.

Reply:
Fet!